src/Filter/FilterData.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Filter;
```

Comment:
Since this class is not a filter, but a DTO, maybe we could use other namespace in order to have a better organization.
 What about `Sonata\AdminBundle\Filter\Model`?